packages/models/src/lib/implementation/schemas.ts

```diff
@@ -168,6 +169,7 @@ export function weightedRefSchema(
     { description },
   );
 }
+export type WeightedRef = z.infer<ReturnType<typeof weightedRefSchema>>; // @TODO why is slug optional? @matej
```

Comment:
Just checked the code - it isn't.

```suggestion
export type WeightedRef = z.infer<ReturnType<typeof weightedRefSchema>>;
```